File: level_1/PYTHON/행렬의 덧셈.py
def solution(arr1, arr2):
    return [[c + d for c, d in zip(a, b)] for a, b in zip(arr1,arr2)]

'''
import numpy as np
def sumMatrix(A,B):
    A=np.array(A)
    B=np.array(B)
    answer=A+B
    return answer.tolist()
    '''

File: level_1/C++/행렬의 덧셈.cpp
#include <vector>

using namespace std;

vector<vector<int>> solution(vector<vector<int>> arr1, vector<vector<int>> arr2) {
	vector<vector<int>> answer = arr1;
	for (int i = 0; i<arr1.size(); i++) {
		for (int z = 0; z<arr1[0].size(); z++) {
			answer[i][z] = arr1[i][z] + arr2[i][z];
		}
	}
	return answer;
}